crowbar_framework/lib/crowbar/logger/formatter.rb

```diff
@@ -18,20 +18,55 @@ module Crowbar
   module Logger
     class Formatter < ::Logger::Formatter
       include ActiveSupport::TaggedLogging::Formatter
+      FORMAT = "%s, [%s#%d] %5s -- %s\n"
+      THREAD_FORMAT = "%s, [%s#%d:%s] %5s -- %s\n"
 
-      def call(severity, timestamp, progname, msg)
-        final_message = if msg.is_a? String
+      def call(severity, time, progname, msg)
+        if (ENV["CROWBAR_THREADS"].to_i || 10) > 1
```

Comment:
> If CROWBAR_THREADS is not set (empty string or nil), or is set to 0, is expected to have 10 threads?

```
irb(main):001:0> "".to_i
=> 0
irb(main):002:0> nil.to_i
=> 0
```

> Also if this is the puma default, I also expect to remove this magic number from the check.

So I'm using the same magic as in config/puma.rb. I'm not really sure if we can do something else to get the default? Any idea?

That being said, my logic is wrong. Should probably be `(ENV["CROWBAR_THREADS"] || 10).to_i`